.evergreen/generate_evergreen_tasks.js

```diff
@@ -728,14 +728,21 @@ BUILD_VARIANTS.push({
 });
 
 BUILD_VARIANTS.push({
-  name: 'no-auth-tests',
+  name: 'ubuntu1804-no-auth-tests',
   display_name: 'No Auth Tests',
-  run_on: 'ubuntu1804-test',
+  run_on: DEFAULT_OS,
```

Comment:
Does having `ubuntu1804` in the variant name provide any benefit?  If we're only running it on one OS, it doesn't seem like we need to specify.  Furthermore, with your change to the `run_on` to use the `DEFAULT_OS`, we now have the possibility for the variant name and the OS to get out of sync if we update the `DEFAULT_OS`.

Reply:
I think per our discussion earlier we can defer the evergreen task cleanup to a later time